src/Application/Redux/StateUtils.js

```diff
@@ -0,0 +1,157 @@
+import ImmutableAsset from './../Models/Asset';
+import { List } from 'immutable';
+import _ from 'underscore';
+
+const PRICE_INCREASE_PERCENTAGE = 7;
+const BASE_CLICK_INCOME = 1;
+
+const createNews = (news, newArticle) => {
+  const copyOfNews = news.slice();
+  copyOfNews.push(newArticle);
+  if (copyOfNews.length > 10) copyOfNews.shift();
+  return copyOfNews;
+};
+
+const unlockAsset = (asset) => {
+  return Object.assign({},
+    asset,
+    {
+      unlocked: true,
+    }
+  );
```

Comment:
this is a good one: all methods that just deal with the fields of a model are very nice to have right on the model.  so under the getters you'd put `toUnlocked()` that does this same thing, and here it's just `return asset.toUnlocked()`